kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/locators/EmptyLocator.kt

```diff
@@ -4,12 +4,18 @@
 
 package kotlinx.kover.gradle.plugin.locators
 
-import kotlinx.kover.gradle.plugin.commons.*
-import kotlinx.kover.gradle.plugin.dsl.internal.KoverProjectExtensionImpl
+import org.gradle.api.Project
 
-
-internal class EmptyLocator : CompilationKitLocator {
-    override fun locate(koverExtension: KoverProjectExtensionImpl): ProjectCompilation {
-        return ProjectCompilation(AppliedKotlinPlugin(null))
+/**
+ * A locator that works if no Kotlin plugins were detected.
+ */
+internal class EmptyLocator(
+    project: Project,
+    listener: CompilationsListenerWrapper
+) {
+    init {
+        project.afterEvaluate {
+            listener.defaultFinalize()
```

Comment:
`afterEvaluate` is a queue with actions, executed in the order they are added to it.
I assumed that Kotlin and Android plugins are not applied within `afterEvaluate` actions. Therefore, this code works as expected.

However, it would be nice to consider the option if plugins are used (for some reason) in `afterEvaluate` action.

Reply:
Ah, I get it. You first call `listener.onApplyPlugin(KotlinPluginType.JVM)` to set the type and only after that set `afterEvaluate` lambda.